pre-docs/inventory_file.md

```diff
@@ -225,10 +225,30 @@ nodes:
           run-as: root
 ```
 
--   **[Generating inventory files](inventory_file_generating.md)**  
```

Comment:
Just FYI, this link in docs proper is auto-generated based on how the publication is set up, so I can't remove it.